installer/create-cluster-eks.sh

```diff
@@ -44,24 +44,13 @@ eksctl create cluster \
   --version 1.14 \
   --region "$AWS_DEFAULT_REGION" \
   --nodegroup-name standard-workers \
-  --node-type t3.medium \
+  --node-type t3.xlarge \
```

Comment:
this is closer to what we have for GKE at the moment, cost is reasonable (more here: https://aws.amazon.com/ec2/pricing/on-demand/).